runtime/rasdump/javadump.cpp

```diff
@@ -969,6 +969,13 @@ JavaCoreDumpWriter::writeEnvironmentSection(void)
 	_OutputStream.writeCharacters(EsBuildVersionString);
 	_OutputStream.writeCharacters("\n");
 
+#if defined(OPENJ9_TAG)
```

Comment:
Looking at the extensions changes, won't `OPENJ9_TAG` always be defined with those changes?  

The common case it would be defined as:
```
#define OPENJ9_TAG
```
with no value.

If that's correct, the `strlen(OPENJ9_TAG)` is equivalent to `strlen()`, which will be a compile error.

Reply:
Its defined as `#define OPENJDK_TAG "@OPENJDK_TAG@"` i.e. `#define OPENJDK_TAG "" when there is no tag.

Reply:
In a previous PR build it was tested (with no tag) and there is no compile error.

Reply:
https://ci.eclipse.org/openj9/job/PullRequest-Sanity-JDK9-linux_ppc-64_cmprssptrs_le-OpenJ9/62/ has passed

Reply:
I missed the quotes in the extensions pr.